blog/cncf-serverlessworkflow-official-recommends-eventmesh-as-runtime-impl.md

```diff
@@ -6,7 +6,7 @@ CNCF [Serverless Workflow](https://serverlessworkflow.io/) defines a vendor-neut
 
 This project is composed of:
 
-* [Specification](specification.md) for defining DSL-based workflows
+* [Specification](https://github.com/serverlessworkflow/specification/blob/main/specification.md#using-functions-for-async-api-service-invocations) for defining DSL-based workflows
```

Comment:
Relative links are preferred. It is not recommanded to redirect users to github doc from apache doc site. Same as below.

Reply:
There is no original specification.md file in that directory, what should I do?

Reply:
👍, This relative link is OK because it is an external document.